web/app/mu-plugins/disallow-indexing.php

```diff
@@ -3,12 +3,24 @@
 Plugin Name:  Disallow Indexing
 Plugin URI:   https://roots.io/bedrock/
 Description:  Disallow indexing of your site on non-production environments.
-Version:      1.0.0
+Version:      2.0.0
 Author:       Roots
 Author URI:   https://roots.io/
+Text Domain:  roots
 License:      MIT License
 */
 
-if (defined('WP_ENV') && WP_ENV !== 'production' && !is_admin()) {
-    add_action('pre_option_blog_public', '__return_zero');
+if (!defined('WP_ENV') || WP_ENV === 'production') {
```

Comment:
What if we just did `Config::define('DISALLOW_INDEXING', WP_ENV !== 'production');` in config/application.php

and then 

```php
if (!defined('DISALLOW_INDEXING') || DISALLOW_INDEXING !== true) {
}
```

It makes the code read a bunch better:

Currently:
> If we are on prod don't disallow indexing

Proposed:
> If we aren't on prod disallow indexing

Additionally this makes the functionality 
- super easy to trace via config/application.php
- does not rely on magic string 'production'
- ability to be overridden per environment with new config system